rho/clicommands.py

```diff
@@ -810,6 +822,10 @@ def _do_command(self):
 
         # removes all inventories ever.
         elif self.options.all:
+            if not os.path.isfile('data/profiles'):
+                print(_("All network profiles removed"))
+                sys.exit(0)
```

Comment:
Why `sys.exit(0)` here? It doesn't match the control flow before the change, and it seems to mean that we have to run this command last or else rho will die prematurely.